content/docs/tasks/traffic-management/debugging-istio/index.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Debugging Istio
```

Comment:
Could we call this Debugging Envoy and Pilot? Istio is more than just Envoy and Pilot and this task doesn't address those other things, so the name is misleading.